spec/types/object.spec.ts

```diff
@@ -146,7 +146,7 @@ describe('object', () => {
         });
 
       const DogSchema = AnimalSchema.extend({
-        type: z.string().openapi({ const: 'dog' }),
+        type: z.string().openapi({ example: 'dog' }),
```

Comment:
`const` is apparently not valid in open api so I've exchanged it with example here